### DISCUSSION TITLE
Slack: use further attachments

### DISCUSSION THREAD

**Discussion by muellerfa** [fastlane/fastlane#21013] [Q&A]
Hey!

I'm trying to make our slack messages a little more comfortable by adding more information to it. So I tried to add some extra attachments like a button in blocks like an example from the Slack webhook documentation which is linked in the fastlane slack example (see the hotel example here: https://api.slack.com/messaging/composing/layouts#attachments).

But I wasn't able to get the blocks and the buttons from the example running correctly. I tried different ways to add the code to the "attachment_properties" field of the slack plugin, but it never shows the blocks or buttons or I had an error running the lane.

Has someone a working example how to get this blocks and buttons working? Or is it not possible while the plugin can't "merge" the blocks correctly to the JSON which will be send?

Thanks!

**Comment by saeidbasirnia** [comment]
@muellerfa did you find a solution for this?

**Reply by muellerfa** [reply]
No. Still no idea how to fix this.

**Reply by saeid** [reply]
I ended up writing a bash script to construct and send the desired slack message and run call it from fastlane.
